mods/ra/rules/ai.yaml

```diff
@@ -42,7 +42,12 @@ Player:
 			stek: 1%
 			fix: 0.1%
 			dome: 10%
+		CapturingActorTypes: e6
+		CapturableActorTypes: mslo, gap, iron, pdox, atek, weap, fact, proc, silo, hpad, afld, powr, apwr, stek, barr, kenn, tent, fix
```

Comment:
You could add `oilb` here and maybe have it prioritize neutral unguarded oil derricks.